Testscripts/Windows/WORKLOAD-HB.ps1

```diff
@@ -278,29 +304,18 @@ echo disk > /sys/power/state
 			}
 		}
 
-		# Send fio 32-job command for 5 min
-		Write-LogInfo "Running fio-2 command"
-		$timeout = New-Timespan -Minutes $maxFIORunWaitMin
-		$sw = [diagnostics.stopwatch]::StartNew()
-		$state = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "bash ./fio2Command.sh" -RunInBackground -runAsSudo
-		Wait-Time -seconds 5
-		while ($sw.elapsed -lt $timeout){
-			Wait-Time -seconds 15
-			$state = Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password "cat /home/$user/state.txt"
-			if ($state -eq "TestCompleted") {
-				Write-LogInfo "Completed fio command execution in the VM $($AllVMData.RoleName) successfully"
-				break
-			} else {
-				Write-LogInfo "$($state): fio command is still running!"
-			}
-		}
-		if ($state -ne "TestCompleted") {
-			throw "fio-2 command is still running after $maxFIORunWaitMin minutes"
-		}
+		Start-WorkLoad "afterhb"
 
 		$testResult = $resultPass
 
-		Copy-RemoteFiles -downloadFrom $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -download -downloadTo $LogDir -files "*.json, *.log" -runAsSudo
+		Copy-RemoteFiles -downloadFrom $AllVMData[0].PublicIP -port $AllVMData[0].SSHPort -username $user -password $password -download -downloadTo $LogDir -files "*.json, *.log" -runAsSudo
+		$work1Output = Get-Content -Path "$LogDir\beforewl.json"
+		$work2Output = Get-Content -Path "$LogDir\afterwl.json"
```

Comment:
after update, you don't have those two beforewl.json and afterwl.json files.

Reply:
I am looking at it now.

Reply:
Could you please point out which lines generated those files?

Reply:
> There are 2 files.
> ![image](https://user-images.githubusercontent.com/23069399/84622014-e775c480-ae90-11ea-97c5-481dca165847.png)

Could you please point out which lines generated those files?

Reply:
The line number 57 in WORKLOAD-HB.ps1